contracts/governance/Staking/WeightedStaking.sol

```diff
@@ -388,7 +388,7 @@ contract WeightedStaking is Checkpoints {
         address account,
         uint256 date,
         uint256 blockNumber
-    ) external view returns (uint96) {
+    ) public view returns (uint96) {
```

Comment:
why public?

Reply:
Fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/c4c43c22e2dc30a936960e01e9a8acd6e2053616)